### PR TITLE
Copyedit updates

### PR DESCRIPTION
**Pull request by KFishner**
I updated to top-level messaging and the supporting text to be more active.

I'm still not happy with the first sentence in this hero text:

>"Develop and deploy applications across large teams and global infrastructure. HashiCorp solves development, operations, and security challenges at scale so organizations can focus on business-critical tasks."

We certainly do enable organizations to "develop and deploy applications across large teams and global infrastructure", but it could probably be more "markety". I'll keep thinking about it, but open to suggestions.

### CODE REVIEW

File: source/atlas.html.erb [hunk 1]
```diff
@@ -160,7 +161,7 @@
             <h1>Open Source at the Core</h1>
             <p>
               Atlas is built on HashiCorp open source
-              tools. Join a massive, open community of contributors
+              tools. Join a dedicated, open community of contributors
```
Comment (pearkes) [inline source/atlas.html.erb]: haha this is way better – massive was a little much.

File: source/index.html.erb [hunk 2]
```diff
@@ -28,10 +27,10 @@
...
           </h2>
           <p>
-            HashiCorp tools are designed and built on core principles to
-            enable a flexible workflow to guide application deployment,
-            maintenance and operations. When followed, this workflow delivers
-            change securely and on time, all while minimizing cost.
+            Securely maintaining global infrastructure while
+            enabling rapid application delivery is challenging. HashiCorp tools
+            provide a flexible workflow so organizations can manage physical, virtual, and
+            containerized workloads across large teams and global infrastructure.
```
Comment (pearkes) [inline source/index.html.erb]: I was trying to get cost minimizing in here too...can we work that in somehow?

**Comment by pearkes** [comment]
One comment but otherwise :+1:

File: source/index.html.erb [hunk 3]
```diff
@@ -41,25 +40,32 @@
...
             <p>
-              Resource usage is defined in code and reacts to changing
-              workloads while maxmimizing density.
+              Reduce infrastructure costs by placing more applications per host and
+              maximizing resource utilization.
+            </p>
+          </div>
+          <div class="why-feature efficient">
+            <span class="graphic"></span>
+            <h2>Flexible</h2>
```
Comment (KFishner) [inline source/index.html.erb]: I added one more "feature" based on feedback. If you're super opposed to this, I can consolidate fast & efficient into one graph

Comment (pearkes) [inline source/index.html.erb]: Nope that's cool.

**Pull request merged**